screens/Settings/SettingScreenController.ts

```diff
@@ -138,14 +139,19 @@ export function useSettingsScreen(props: RootRouteProps) {
         )
       ),
 
+    RECEIVE_CARD: () => {
+      props.navigation.navigate('Request');
+      setIsVisible(false);
+    },
+
     TOGGLE_BIOMETRIC: (enable: boolean) =>
       settingsService.send(SettingsEvents.TOGGLE_BIOMETRIC_UNLOCK(enable)),
 
     LOGOUT: () => {
       setIsVisible(false);
       const navigate = () => {
         authService.send(AuthEvents.LOGOUT());
-        props.navigation.navigate('Welcome');
+        props.navigation.navigate('Request');
```

Comment:
With this change will the Logout button open the Request-VC screen?

Reply:
In the trial and error process, we checked like this. However, there is no need of this line in code & it would never affect anything. Still, logout navigates to the 'Welcome Screen' Anyhow I removed this unused part.